extensions/desktop/common/desktop-exports

```diff
@@ -173,22 +183,18 @@ for ((i = 0; i < ${#XDG_SPECIAL_DIRS_PATHS[@]}; i++)); do
 done
 
 # If needs XDG update and xdg-user-dirs-update exists in $PATH, run it
-if [ $needs_xdg_update = true ] && command -v xdg-user-dirs-update >/dev/null; then
+if [ "$needs_xdg_update" = true ] && command -v xdg-user-dirs-update >/dev/null; then
   xdg-user-dirs-update
   update_xdg_dirs_values
 fi
 
 # Create links for user-dirs.dirs
-if [ $needs_xdg_links = true ]; then
+if [ "$needs_xdg_links" = true ]; then
   for ((i = 0; i < ${#XDG_SPECIAL_DIRS_PATHS[@]}; i++)); do
-    b="$(realpath "${XDG_SPECIAL_DIRS_PATHS[$i]}" --relative-to="$HOME")"
+    b=$(realpath "${XDG_SPECIAL_DIRS_PATHS[$i]}" --relative-to="$HOME")
```

Comment:
You know, I've always quoted these. TIL, all of those quotes were wasted! :D

But it probably should be done consistently (e.g. vs. usage in `is_subpath()`)